src/auth/src/credentials/idtoken/user_account.rs

```diff
@@ -112,6 +113,23 @@ impl Builder {
     ///
     /// Any value provided here overrides a `token_uri` value from the input `authorized_user` JSON.
     /// Defaults to `https://oauth2.googleapis.com/token` if not specified here or in the `authorized_user` JSON.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use google_cloud_auth::credentials::idtoken;
+    /// # use serde_json::json;
+    /// # let authorized_user = json!({
+    /// #     "client_id": "YOUR_CLIENT_ID.apps.googleusercontent.com",
+    /// #     "client_secret": "YOUR_CLIENT_SECRET",
+    /// #     "refresh_token": "YOUR_REFRESH_TOKEN",
+    /// #     "type": "authorized_user",
+    /// # });
```

Comment:
Adding all these details and then hiding them makes no sense to me.  Either say something like `let authorized_user: serde_json::Value = hidden_function();` or show the details.